### PR TITLE
User Guide update

### PR DESCRIPTION
**Pull request by TOsmanov**

### CODE REVIEW

**Comment by nmanovic** [comment]
@TOsmanov , please fix codacy issues.

File: cvat/apps/documentation/user_guide.md [hunk 1]
```diff
@@ -168,53 +175,86 @@ Go to the [Django administration panel](http://localhost:8080/admin). There you
...
+   1. Open ``Actions`` menu.  
+   1. Change issue tracker or open issue tracker if it is specified.
+   1. Change labels.You can add new labels or add attributes for the existing labels in the Raw mode or the Constructor mode. By clicking ``Copy`` you will copy the labels to the clipboard.
+   1. Assigned to — is used to assign a task to a person. Start typing an assignee’s name and/or choose the right person out of the dropdown list.
+
+   1. ``Jobs`` — is a list of all jobs for a particular task. Here you can find the next data:
+      - Jobs name whit a hyperlink to it.
+      - Frames — the frame interval. 
+      - A status of the job. The status is specified by the user in the menu inside the job. There are three types of status: annotation, validation or completed. The status of the job is changes the progress bar of the task.
+      - Started on —
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md]: Missing definitation

**Comment by nmanovic** [comment]
@TOsmanov , I see the following plan:
- Please fix codacy issues
- Please fix some found missing definitions
- After that I will merge the PR.
- Let's have a meeting one a week to describe what should be done next (e.g. model manager, task update a separate topic)
- Next patches I will suggest sending by a topic (e.g. model manager, automatic annotation). Thus it will be easy to review.

**Comment by TOsmanov** [comment]
@nmanovic, I suggest the following user guide development plan:

- Model Manager:
1. Change images
1. Add page description "/ models"
1. Add page description "/ models/create"

- Auto annotations:
1. Change images
1.  Delete text about create model
1. More detailed text on the correspondence of model labels and task labels
1. Replacing the text “manual merge” with “reid merge”

- Synchronization of tasks with storage:
1. Change images
1. Change part of the text to reduce the number of steps

**Pull request merged**